.github/workflows/infra-meeting-release.yaml

```
@@ -0,0 +1,117 @@
+          result-encoding: string
+          script: |
+            const getMilestoneAsMarkdown = async function(milestone, milestoneName, issuesState) {
+              const opts = github.rest.issues.listForRepo.endpoint.merge({
+                ...context.issue,
+                milestone,
+                state: issuesState
+              })
+              const issues = await github.paginate(opts)
+
+              let markdown = ''
+              let category = 'Done'
+
+              // There should not be any closed issue in the 'next' milestone, only 'Done' ones
+              if (issuesState != 'closed') {
+                category = (milestoneName != 'next') ? 'Work In Progress' : 'New/Important'
+              }
+
+              if (issues.length > 0) {
+                markdown = `* [${category}](${context.payload.repository.html_url}/milestone/${milestone}?closed=1):`
+                for (const issue of issues) {
+                  markdown = markdown.concat("\r\n").concat(`  * [${issue.title}](${issue.html_url})`)
+                }
+              }
+
+              return markdown
+            }
+
+            done = await getMilestoneAsMarkdown(context.payload.inputs.milestone_id, context.payload.inputs.milestone_name, 'open')
+            wip = await getMilestoneAsMarkdown(context.payload.inputs.milestone_id, context.payload.inputs.milestone_name, 'closed')
+            next = await getMilestoneAsMarkdown(context.payload.inputs.next_milestone_id, context.payload.inputs.next_milestone_name, 'open')
+
+            return `Markdown for the infra team sync meeting notes preparation:
+            <pre>
+            ${done}
+
+            ${wip}
+
+            ${next}
+            </pre>
+
+            <details><summary>Preview:</summary>
+
+            ${done}
+
+            ${wip}
+
+            ${next}
+
+            </details>
+
+            Generated from the ["${context.payload.inputs.milestone_name}"](${context.payload.repository.html_url}/milestone/${context.payload.inputs.milestone_id}) and the ["${context.payload.inputs.next_milestone_name}"](${context.payload.repository.html_url}/milestone/${context.payload.inputs.next_milestone_id}) milestones.`
+
+      - name: "Create release"
+        id: create_release
+        uses: "actions/github-script@v6"
+        env:
+          CURRENT_DATE: ${{ steps.date.outputs.date }}
+          RELEASE_BODY: ${{steps.milestones_as_markdown.outputs.result}}
+        with:
+          script: |
+            name = `infra-team-sync-${process.env.CURRENT_DATE}`
+            tag = `${name}_${context.runNumber}`
+
+            try {
+              await github.rest.repos.createRelease({
+                name: name,
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                body: process.env.RELEASE_BODY,
+                tag_name: tag,
+                draft: true,
+                generate_release_notes: true,
+                prerelease: false,
```

```suggestion
                discussion_category_name: 'General',
                prerelease: false,
```
Not added as it doesn't work if draft: true, to be added via gha on release publication (+milestone closing/creation?)